tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetCore5IastTests.cs

```diff
@@ -587,6 +585,75 @@ await VerifyHelper.VerifySpans(spansFiltered, settings)
                             .UseFileName(filename)
                             .DisableRequireUniquePrefix();
     }
+
+    // We should exclude some headers to prevent false positives:
```

Comment:
I'd put this comment in the analyzer function. It's not clear why the logic is that way. This makes things clearer

Reply:
I have pasted the comment in the analyzer